projects/hslayers/src/components/layout/layout.service.ts

```diff
@@ -484,11 +484,11 @@ export class HsLayoutService {
     });
   }
 
-  createPanel(panelComponent: Type<any>, app: string, data?: any): void {
+  createPanel(panelComponent: Type<any>, app?: string, data?: any): void {
```

Comment:
```suggestion
createPanel(panelComponent: Type<any>, app = 'default', data?: any): void {
```

Reply:
Is not important ere though, since we have the same thing in PanelContainerService->create method